doc/modules/mod_inbox.md

```diff
@@ -5,19 +5,51 @@ To use it, enable mod\_inbox in the config file.
 
 ### Options
 
-* **backend** (atom, default: `rdbms`) - Database backend to use. For now, only `rdbms` is supported.
-* **reset_markers** (list, default: `[displayed]`) - List of atom chat markers that when sent, will reset the unread message counter for a conversation.
+#### `modules.mod_inbox.backend`
+* **Syntax:** string
+* **Default:** `rdbms`
+* **Example:** `backend = rdbms`
+
+Database backend to use. For now, only `rdbms` is supported.
+
+#### `modules.mod_inbox.reset_markers`
+* **Syntax:** array of strings
+* **Default:** `["displayed"]`
+* **Example:** `reset_markers = ["received"]`
+
+List of atom chat markers that when sent, will reset the unread message counter for a conversation.
```

Comment:
Should we use the word `atom` in a TOML documentation at all?

Reply:
idk honestly, but what do you propose to use instead?

Reply:
I suppose, atoms are strings now.

Reply:
Exactly, there is no need to specify the internal format, be it atom, string or a binary

Reply:
So I'll go just for `chat markers` as the strings are already enforced by syntax.